05-JWT-Basics/starter/errors/custom-error.js

```diff
@@ -1,7 +1,6 @@
 class CustomAPIError extends Error {
   constructor(message, statusCode) {
     super(message)
-    this.statusCode = statusCode
```

Comment:
You can keep this in custom api error and just do super(message, <desired status code>) in the error classes that extend it

Reply:
To clarify a bit more on this are you suggesting having the 
` this.statusCode = statusCode`
in the module BadRequest since it would extend from the parent custom-error module?